Tribler/Test/Core/test_sqlitecachedbhandler.py

```diff
@@ -59,7 +59,6 @@ def setUp(self):
         self.sqlitedb.initialize()
         self.session.sqlite_db = self.sqlitedb
 
-    @blocking_call_on_reactor_thread
     def tearDown(self):
         self.sqlitedb.close()
```

Comment:
The `blocking_call_on_reactor_thread` was a mistake and shouldn't be there. Closing the sqlite database is apparently something that can happen on the main thread: we are also doing it when closing Tribler (both when running from command line and when running the tests).